src/main/java/com/statsig/androidsdk/StatsigPendingRequests.kt

```diff
@@ -3,10 +3,10 @@ package com.statsig.androidsdk
 import com.google.gson.annotations.SerializedName
 
 data class StatsigPendingRequests(
-    @SerializedName("requests") val requests: ArrayList<StatsigOfflineRequest>
+    @SerializedName("requests") val requests: List<StatsigOfflineRequest>
```

Comment:
Not sure if the requests should be mutable, so modified this to be safe.

Reply:
Should only be created once, at startup, for now.